### PR TITLE
Refactor config, Add makefile

### PR DESCRIPTION
**Pull request by caudaganesh**

### CODE REVIEW

File: config/proxy.go [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	})
+
+	if err != nil {
+		return err
+	}
+
+	return determineNext(s.next, p)
+}
+
+type SetTowFAStation struct {
```
Comment (adzimzf) [inline config/proxy.go]: ada typo dikit nesh

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+install :
+	@go install github.com/adzimzf/tpot
```
Comment (adzimzf) [inline Makefile:2]: @caudaganesh buat release nya pake `goreleaser` nesh.
kalo di tambahin make file nya juga lumayan membantu nih
```shell scrpt
goreleaser --rm-dist
```

**Review by adzimzf — CHANGES_REQUESTED** [review]
I have some request changes nesh

File: Makefile (hunk 2 above)
Comment (caudaganesh) [reply Makefile:2]: next deh ini yak wkwk, gw kudu pelajarin dulu

**Review by adzimzf — APPROVED** [review]

**Pull request merged**